test/integration/generator/content/schedule/part3/index-with-get-summary-components.test.js

```diff
@@ -0,0 +1,84 @@
+jest.mock('../../../../../../app/generator/content/schedule/part3/get-table')
+const { getTable } = require('../../../../../../app/generator/content/schedule/part3/get-table')
+
+const part3 = require('../../../../../../app/generator/content/schedule/part3')
+
+const { topUpSchedule, reductionSchedule, reductionZeroSchedule, negativeRemainingAmountSchedule } = require('../../../../../mocks/mock-schedule')
+
+describe('schedule part 3 integration with getSummary components', () => {
```

Comment:
Test file name doesn't refer to a file.

Should be schedule/part3/index -- consistent w/ other modules and their tests

This describe name can just be part 3

Reply:
I would also argue these tests are unit not integration because we are mocking a func call w/in and don't care what these return.